pkg/printer/workload_source_printer.go

```diff
@@ -157,3 +159,50 @@ func WorkloadSourceGitPrinter(w io.Writer, workload *cartov1alpha1.Workload) err
 
 	return tablePrinter.PrintObj(workload, w)
 }
+
+func WorkloadResourcesPrinter(w io.Writer, workload *cartov1alpha1.Workload) error {
+	printResourceInfoRow := func(resource *cartov1alpha1.RealizedResource, _ table.PrintOptions) ([]metav1beta1.TableRow, error) {
+		var lastTransitionTime string
+		var ready string
+		for _, r := range resource.Conditions {
+			if r.Type == cartov1alpha1.ResourceReady {
```

Comment:
You could consider adding a helper function called `GetCondition` in  package cartov1alpha1, and make this just a caller. This will simplify the table logic and  cartov1alpha1 can do the heavy lifting.

Reply:
I used  [apimachinery/meta](https://github.com/kubernetes/apimachinery/blob/master/pkg/api/met) function instead of writing a custom one because this one does what we need. Don't need to duplicate it, what do you think? @shashwathi